src/main/webapp/app/controllers/submission/adminSubmissionViewController.js

```diff
@@ -377,7 +366,32 @@ vireo.controller("AdminSubmissionViewController", function ($anchorScroll, $cont
             fieldValue.refresh();
         };
 
-        $scope.submitAddFile = function () {
+        $scope.resetFileData = function (addFileData) {
+            $scope.closeModal();
+            $scope.errorMessage = "";
+            $scope.removeFiles();
+            
+            addFileData.uploading = false;
+            addFileData.recipientEmail = '';
+            addFileData.recipientEmails = userSettings.attachment_email_student_by_default === "true" ? [new EmailRecipient({
+                name: "Submitter",
+                type: EmailRecipientType.SUBMITTER,
+                data: "Submitter"
+              })] : [];
+            addFileData.ccRecipientEmail = '';
+            addFileData.ccRecipientEmails = userSettings.attachment_cc_student_advisor_by_default === "true" ? $scope.submission.getContactEmails() : [];
+            addFileData.sendEmailToRecipient = (userSettings.attachment_email_student_by_default === "true") || (userSettings.attachment_cc_student_advisor_by_default === "true");
+            addFileData.sendEmailToCCRecipient = userSettings.attachment_cc_student_advisor_by_default === "true";
+            addFileData.subject = "";
+            addFileData.message = "";
+            addFileData.selectedTemplate = $scope.emailTemplates[0];
+            addFileData.needsCorrection = userSettings.attachment_flag_submission_as_needs_corrections_by_default === "true";
+
+        };
+
+        $scope.resetFileData($scope.addFileData);
```

Comment:
Couldn't `resetFileData()` just access `$scope.addFileData` directly? 
For example, changing line 374 from `addFileData.uploading = false;` into `$scope.addFileData.uploading = false;`?